src/components/team/MemberCard.js

```diff
@@ -6,10 +6,10 @@ function MemberCard(props) {
 
   return (
     <Card className="member-card" bordered={false}>
-      <div className='image-container'>
+      {/* <div className='image-container'>
         {(!member.img) ? (<Avatar size={64} icon="user" />): (<img src = {member.img} />)}
-      </div>
-      <p className="small">{member.name}</p>
+      </div> */}
+      <p className="small">{member.user_full_name}</p>
```

Comment:
If you change this like so I'll update the backend to concatenate the first_name and last_name correctly, and then the name will load.

```suggestion
      <p className="small">{member.user_name}</p>
```